lib/spring/test/application.rb

```diff
@@ -94,7 +94,7 @@ def run(command, opts = {})
         Bundler.with_clean_env do
           Process.spawn(
             env,
-            command.to_s,
+            "bundle exec #{command}",
```

Comment:
> We are already setting GEM_PATH and GEM_HOME in the env of the spawned process, so I don't understand why it would be an issue.

Yes, it shouldn't be an issues and it isn't, at least locally. However, tests on travis fail with conflicting bundler version. For *some* reason [travis installs bundler 1.7.6](https://travis-ci.org/sonalkr132/spring/jobs/96552775#L101) at first and later it ends up conflicting with generated rails app bundler. Same thing happened in last two attempts of this fix ( #293, #373).